CONTRIBUTING.md

```diff
@@ -57,21 +57,22 @@ To set up `pysatModels` for local development:
    ``test_``.
 
 4. When you're done making changes, run all the checks to ensure that nothing
-   is broken on your local system::
+   is broken on your local system.  You may need to install pytest and
+   pytest-flake8 first. ::
```

Comment:
We need to specify which directory we're supposed to be running this command from.  Also, it doesn't work for me.  I had to preface this with `python3 -m` and run it from the test directory.

Recommend using quotes: `pytest` and `pytest-flake8`, so that the user is less likely to confuse `flake8` with `pytest-flake8`.